frontend/dashboard/pages/Dashboard/Dashboard.module.css

```diff
@@ -9,6 +9,11 @@
   margin-bottom: var(--fds-spacing-10);
 }
 
+.searchContainer {
```

Comment:
I suggest changing the CSS class names to `search`, since they are applied to the components themselves, not their containers.
```suggestion
.search {
```